Remove alt text from Rango logo to hide from assistive technologies

bacall213 opened the pull request:
Rango's logo on the setting page has the alt-text "Rango logo" but the image is decorative in nature and doesn't provide any information so it should be hidden from assistive technologies: https://www.w3.org/WAI/tutorials/images/decorative/.

bacall213 committed:
Remove alt text from Rango logo to hide from assistive technologies like screen readers

Rango's logo on the setting page has the alt-text "Rango logo" but the image is decorative in nature and doesn't provide any information so it should be hidden from assistive technologies: https://www.w3.org/WAI/tutorials/images/decorative/.

david-tejada commented:
I am getting a prettier error in the tests. Could you remove the `{" "}` in that line? I am not sure what was doing there anyway.

bacall213 committed:
Update App.tsx - Remove extra curly braces after logo

Remove extra curly braces after logo. Not sure what they do, but prettier was apparently throwing an error after the alt text was removed for the logo.

bacall213 commented:
I'm not sure what the extra {" "} does either :) I've removed it.
